From b722e68c001db5bd41644057cdf6ef4ab04a3217 Mon Sep 17 00:00:00 2001
From: Sam Hughes <sam@samuelhughes.com>
Date: Wed, 7 Jun 2023 19:10:14 -0700
Subject: [PATCH 2/2] Add stdint.h in utils.hpp too

Another include-what-you-use fix, the last failure seen prior
to adding stdint.h to errors.hpp.
---
 src/utils.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/utils.hpp b/src/utils.hpp
index 53a1295aea5f..32db0632f2c0 100644
--- a/src/utils.hpp
+++ b/src/utils.hpp
@@ -2,6 +2,7 @@
 #ifndef UTILS_HPP_
 #define UTILS_HPP_
 
+#include <stdint.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <time.h>
-- 
2.41.0

